packages/cli/test/integration/generators/model.integration.js

```diff
@@ -65,7 +65,7 @@ describe('lb4 model integration', () => {
       testUtils
         .executeGenerator(generator)
         .inDir(SANDBOX_PATH, () =>
-          testUtils.givenLBProject(SANDBOX_PATH, {excludeKeyword: true}),
+          testUtils.givenLBProject(SANDBOX_PATH, {excludeKeyword: false}),
```

Comment:
for my knowledge, what does this change entail?

Reply:
We now fail fast. With the `excludeKeyword` is set `true` (no `loopback` keyword in package.json), the first error is surfaced. Before this change, the 2nd error overrides the 1st one.